ci-operator/step-registry/api-job/k8sgpt/api-job-k8sgpt-commands.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+openai_token=$(cat "/var/run/vault/tests-private-account/openai-token")
+
+k8sgpt version
+k8sgpt auth add -m gpt-3.5-turbo -b openai -p ${openai_token} | tee ${ARTIFACT_DIR}/result || true
```

Comment:
@jianzhangbjz better to add `set +x` before this line, like
```shell
set +x
k8sgpt auth add -m gpt-3.5-turbo -b openai -p ${openai_token} | tee ${ARTIFACT_DIR}/result || true
set -x
```
I am fear the token is printed on the terminal with command.

Reply:
Good catch! Thanks!